### PR TITLE
More stuff

### PR DESCRIPTION
**Pull request by qznc**
The commits makefile, space-after-return, and noDoubleNewline are quite trivial.

The comment-commit is more complex. Look into tests/comments.d.ref to get an impression of the results.

I also added some tests I have, which currently fail. Some things are obvious bugs, but with others I am not sure what the desired behavior actually is.

### CODE REVIEW

File: src/dfmt.d [hunk 1]
```diff
@@ -115,8 +115,25 @@ private:
...
+                    output.write(" ");
+            }
             writeToken();
-            newline();
+            if (i >= tokens.length-1)
+                newline();
+            else if (tokens[i+1].line > tokens[i].line)
+                newline();
+            else if (tokens[i+1].type != tok!"{")
+                output.write(" ");
```
Comment (Hackerpilot) [inline src/dfmt.d]: output.write doesn't increase the counter for the length of the current line the way that write() does. Is this intentional?

File: tests/guessnumber.d.ref [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        {
+            writeln("  Out of range!");
+            continue;
+        }
+        if (answer == target)
+        {
+            writeln("  Well guessed.");
+            break;
+        }
+        writeln(answer < target ? "  Too low.":"  Too high.");
```
Comment (Hackerpilot) [inline tests/guessnumber.d.ref]: At some point I need to fix this, probably by asking the AST where the attribute declarations and labels are.

File: src/dfmt.d (hunk 1 above)
Comment (qznc) [inline src/dfmt.d]: No. Will fix it.

**Comment by qznc** [comment]
output.write => write.

**Pull request merged**